recipes/mpir/bld.bat

```diff
@@ -0,0 +1,19 @@
+cd build.vc14
```

Comment:
The easiest way might be checking the Python version (though this may break and need to be fixed in the future). Though maybe @msarahan knows some better way to access this.